ansible/roles/products/teleop/server/desktop-icons/templates/scripts/show_documentation.j2

```diff
@@ -0,0 +1,7 @@
+#jinja2: trim_blocks:False
```

Comment:
I was thinking, maybe would be better to start hosting the documentation right away everytime the container is started (add it to command of the dockerfile) and use the icon just to open firefox and access the link? Unless we dont always want this server to be running, just for the aurora setup for example? No suggesting either or, just brainstorming here